File: test/aidbox_sdk/snapshots/typescripthl7-fhir-r4-core/Schedule.ts
import { Period } from "./Period";
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type Schedule = DomainResource & {
    actor: Reference[];
    active?: boolean;
    comment?: string;
    specialty?: CodeableConcept[];
    identifier?: Identifier[];
    serviceType?: CodeableConcept[];
    planningHorizon?: Period;
    serviceCategory?: CodeableConcept[];
};

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Schedule.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Period
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class Schedule(DomainResource):
    actor: list[Reference] = field(default_factory=list)
    active: Optional[bool] = None
    comment: Optional[str] = None
    specialty: Optional[List[CodeableConcept]] = field(default_factory=list)
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    service_type: Optional[List[CodeableConcept]] = field(default_factory=list)
    planning_horizon: Optional[Period] = None
    service_category: Optional[List[CodeableConcept]] = field(default_factory=list)